.github/workflows/ci-build.yml

```diff
@@ -1,22 +1,112 @@
 name: CI Build
 on: 
+  workflow_dispatch:
   pull_request:
   # By default, the pull_request event type is not triggered when a PR is merged into master
   push:
     branches:
       - master
+      - use-local-ICAT-in-CI-runs-#641
```

Comment:
Is this still needed? Was it being used for testing just on this branch?

Reply:
Ah yes, removed in https://github.com/ral-facilities/datagateway/pull/753/commits/fd1ce9bed3130d8b505614cbd37a5be20ad4e318